src/MomentumActuatorSrcElemKernel.C

```diff
@@ -0,0 +1,109 @@
+/*------------------------------------------------------------------------*/
```

Comment:
This looks good.  It would be nice if it was more generic, like taking two vector-type fields instead of specifically the actuator field.

Reply:
Cool... so the names of the fields would be inputs to the constructor?